src/scripts/main.js

```diff
@@ -1,3 +1,90 @@
 'use strict';
 
-// write code here
+const tHead = document.querySelector('thead');
+const tBody = document.querySelector('tbody');
+const rows = [...tBody.children];
+let reverse = false;
+
+function handleEvent(e) {
+  let newList = [...rows];
+
+  if (reverse) {
+    reverse = false;
```

Comment:
I think it's better to move this check inside cases.
For now, you have two different switch...cases, but everything can be done with only one.
Let's try to optimize your code